docs/modules/commons/pages/expressions.adoc

```diff
@@ -1086,6 +1086,25 @@ Then `#{extractQueryFromUrl(file://server/folder/data.xml?id=123)}` is = `id=123
 Then `#{extractQueryFromUrl(https://docs.vividus.dev/vividus/latest?page=2)}` is = `page=2`
 ----
 
+=== `extractValueFromUrl`
+
+Extracts decoded value from the given URL. A value is everything after the last '/'
```

Comment:
https://example.com => example.com?

Reply:
a bit strange expression :)

Reply:
We don't need to introduce expression for this case, regex-based one or eval-expression can be used for this rare case

Reply:
removed